app/src/modelProviders/fine-tuned/getCompletion-2.ts

```diff
@@ -129,7 +129,7 @@ async function getModalCompletion(
   return {
     id: resp.id,
     object: "chat.completion",
-    created: Date.now(),
+    created: Math.round(Date.now() / 1000),
```

Comment:
As part of testing this change, I noticed that OpenAI returns the current date in seconds, not milliseconds.